apps/web/src/server/views/applications/case-documentation/documentation-folder.njk

```diff
@@ -10,6 +10,20 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/tag/macro.njk" import govukTag %}
 
+{% block errorSummary %}
+    {% if errors %}
+      <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
```

Comment:
copied identical from the global app.layout but with the 'grid-column-full' class rather than the 'two-thirds-width'